### ISSUE TITLE
Unclear error if one of the parameters is missing

### ISSUE THREAD

**Issue by NinoLipartiia4ire** [near/near-cli-rs#99]
If instead of the correct command
```
./near-cli 
    add contract-code 
    network testnet 
    account 1.cli2.testnet 
    contract-file '~/projects/near/contract/res/contract.wasm' 
    no-initialize 
    sign-with-keychain
```
you run a command which misses a parameter, for example:
```
./near-cli 
    add contract-code 
    account 1.cli2.testnet 
    contract-file '~/projects/near/contract/res/contract.wasm' 
    no-initialize 
    sign-with-keychain
```
or 
```
./near-cli 
    add contract-code 
    network testnet 
    contract-file '~/projects/near/contract/res/contract.wasm' 
    no-initialize 
    sign-with-keychain
```
The error is shown:
```
Error: 
   0: add command or near-cli-add extension does not exist

Location:
   src/common.rs:1221
```
It is quite unclear and doesn't give any information about what parameters are missing.

**Comment by frol** [comment]
Yeah, it is a deficiency of the extensions catching code. We should somehow check if the error stems from finding the top-level command and only if that is the case, search for an extension and otherwise print the original error.

**Comment by frol** [comment]
FYI, @FroVolod is currently working on it

**Comment by frol** [comment]
Resolved in #107
